sympy/vector/functions.py

```diff
@@ -474,3 +474,54 @@ def _path(from_object, to_object):
         from_path.append(other_path[i])
         i -= 1
     return index, from_path
+
+def orthogonalize(vlist, orthonormal=False):
+    """
+    Takes a set of independent vectors and orthogonalizes them
+    using the Gram - Schmidt process. Returns a set of
+    orthogonal or orthonormal vectors.
+
+    Parameters
+    ==========
+
+    vlist : list of independent vectors to be made orthogonal.
```

Comment:
+1. Since you have only one required parameter and one optional, specify them as if. Helps in IDE environments for debugging too. And calling vlist as an iterable makes sense too, since the underlying code would work for any sequence.